src/Flex.php

```diff
@@ -594,6 +605,17 @@ private function fetchRecipes(): array
             }
 
             if (isset($manifests[$name])) {
+                if (isset($manifests[$name]['manifest']['conflict'])) {
+                    foreach ($manifests[$name]['manifest']['conflict'] as $conflictingPackageName => $constraint) {
+                        if ($conflictingPackage = $this->composer->getLocker()->getLockedRepository()->findPackage($conflictingPackageName, $constraint)) {
+                            $conflicts[$name][$conflictingPackageName] = [$conflictingPackage->getPrettyVersion(), $constraint];
+                        }
+                    }
+                    unset($manifests[$name]);
+                    $this->lock->remove($name);
```

Comment:
really? feels unrelated to me.

Reply:
If there is a conflict we need to revert that call, https://github.com/symfony/flex/pull/464/files#diff-46b78198ae7ea525f04268205dd782c3R602.

Reply:
Got ya, fixed, thx